components/phpunit_bridge.rst

```diff
@@ -350,6 +350,16 @@ the compiling and warming up of the container:
 
     The ``--deprecations`` option was introduced in Symfony 5.1.
 
+Log Deprecations
+~~~~~~~~~~~~~~~~
+
+For turning the verbose output off and write it to a log file instead you can use
+``SYMFONY_DEPRECATIONS_HELPER=logFile=/var/logdeprecations.log``.
```

Comment:
why not

```suggestion
``SYMFONY_DEPRECATIONS_HELPER=logFile=/path/to/project/var/log/phpunit_deprecations.log``.
```

?

Reply:
@noniagriconomie Thanks, I changed it to `/path/deprecations.log` (makes it more clear that the path can be anything; with phpunit_deprecations.log I know what you mean but it doesn't sound correct to me because the file won't contain PHPUnit deprecations..)